lib/rtoolsHCK.rb

```diff
@@ -176,7 +176,18 @@ def validate_init_opts(init_opts)
   end
 
   def start_studio_service(service_name)
-    run("Start-Service #{service_name}")
+    retries = 0
+    timeout = 30
+    begin
+      run("Start-Service #{service_name}")
+    rescue WinrmPSRunError => e
+      raise e unless (retries += 1) < 3
+
+      logger('warn', "#{e.message} Retrying in #{timeout} seconds...")
+      sleep(timeout)
+
+      retry
+    end
   end
```

Comment:
Move timeout and `3`(max retry count) to global constant

No need for an extra begin/end block just use `retries ||= 0`